Lib/Protocols/IdSSLOpenSSLHeaders.pas

```diff
@@ -20033,49 +20097,49 @@ function GetCryptLibHandle : TIdLibHandle;
+  {CH fn__ossl_old_des_fcrypt = '_ossl_old_des_fcrypt';} {Do not localize}
+  {CH fn__ossl_old_des_crypt = '_ossl_old_des_crypt';} {Do not localize}
     {$IFNDEF PERL5}
       {$IFNDEF NeXT}
-  {CH fn__ossl_old_crypt = '_ossl_old_crypt'; {Do not localize}
+  {CH fn__ossl_old_crypt = '_ossl_old_crypt';} {Do not localize}
       {$ENDIF}
     {$ENDIF}
-  {CH fn__ossl_old_des_ofb_encrypt = '_ossl_old_des_ofb_encrypt'; {Do not localize}
-  {CH fn__ossl_old_des_pcbc_encrypt = '_ossl_old_des_pcbc_encrypt'; {Do not localize}
-  {CH fn__ossl_old_des_quad_cksum = '_ossl_old_des_quad_cksum'; {Do not localize}
-  {CH fn__ossl_old_des_random_seed = '_ossl_old_des_random_seed'; {Do not localize}
-  {CH fn__ossl_old_des_random_key = '_ossl_old_des_random_key'; {Do not localize}
-  {CH fn__ossl_old_des_read_password = '_ossl_old_des_read_password'; {Do not localize}
-  {CH fn__ossl_old_des_read_2passwords = '_ossl_old_des_read_2passwords'; {Do not localize}
+  {CH fn__ossl_old_des_ofb_encrypt = '_ossl_old_des_ofb_encrypt';} {Do not localize}
+  {CH fn__ossl_old_des_pcbc_encrypt = '_ossl_old_des_pcbc_encrypt';} {Do not localize}
+  {CH fn__ossl_old_des_quad_cksum = '_ossl_old_des_quad_cksum';} {Do not localize}
+  {CH fn__ossl_old_des_random_seed = '_ossl_old_des_random_seed';} {Do not localize}
+  {CH fn__ossl_old_des_random_key = '_ossl_old_des_random_key';} {Do not localize}
+  {CH fn__ossl_old_des_read_password = '_ossl_old_des_read_password';} {Do not localize}
+  {CH fn__ossl_old_des_read_2passwords = '_ossl_old_des_read_2passwords';} {Do not localize}
   fn__ossl_old_des_set_odd_parity = '_ossl_old_des_set_odd_parity'; {Do not localize}
-  {CH fn__ossl_old_des_is_weak_key = '_ossl_old_des_is_weak_key'; {Do not localize}
+  {CH fn__ossl_old_des_is_weak_key = '_ossl_old_des_is_weak_key';} {Do not localize}
   fn__ossl_old_des_set_key = '_ossl_old_des_set_key'; {Do not localize}
-  {CH fn__ossl_old_des_key_sched = '_ossl_old_des_key_sched'; {Do not localize}
-  {CH fn__ossl_old_des_string_to_key = '_ossl_old_des_string_to_key'; {Do not localize}
-  {CH fn__ossl_old_des_string_to_2keys = '_ossl_old_des_string_to_2keys'; {Do not localize}
-  {CH fn__ossl_old_des_cfb64_encrypt = '_ossl_old_des_cfb64_encrypt'; {Do not localize}
-  {CH fn__ossl_old_des_ofb64_encrypt = '_ossl_old_des_ofb64_encrypt'; {Do not localize}
-  {CH fn__ossl_096_des_random_seed = '_ossl_096_des_random_seed'; {Do not localize}
+  {CH fn__ossl_old_des_key_sched = '_ossl_old_des_key_sched';} {Do not localize}
+  {CH fn__ossl_old_des_string_to_key = '_ossl_old_des_string_to_key';} {Do not localize}
+  {CH fn__ossl_old_des_string_to_2keys = '_ossl_old_des_string_to_2keys';} {Do not localize}
+  {CH fn__ossl_old_des_cfb64_encrypt = '_ossl_old_des_cfb64_encrypt';} {Do not localize}
+  {CH fn__ossl_old_des_ofb64_encrypt = '_ossl_old_des_ofb64_encrypt';} {Do not localize}
+  {CH fn__ossl_096_des_random_seed = '_ossl_096_des_random_seed';} {Do not localize}
   {$ENDIF}

```

Comment:
I have incorporated these changes into the `master` code. Please resync your PR.